src/constants.ts

```diff
@@ -6,25 +6,32 @@ export const VERSION = packageJson.version;
 export const DEFAULT_QUERY_LIMIT = 100;
 
 // This is used for sharing evals.
-export const SHARE_API_BASE_URL =
-  // TODO(ian): Backwards compatibility, 2024-04-01
-  getEnvString('NEXT_PUBLIC_PROMPTFOO_REMOTE_API_BASE_URL') ||
-  getEnvString('NEXT_PUBLIC_PROMPTFOO_BASE_URL') ||
-  getEnvString('PROMPTFOO_REMOTE_API_BASE_URL') ||
-  `https://api.promptfoo.app`;
-
-export const DEFAULT_SHARE_VIEW_BASE_URL = getEnvString(
-  'PROMPTFOO_SHARING_APP_BASE_URL',
-  `https://promptfoo.app`,
-);
+export function getShareApiBaseUrl(): string {
+  return (
+    // TODO(ian): Backwards compatibility, 2024-04-01
+    getEnvString('NEXT_PUBLIC_PROMPTFOO_REMOTE_API_BASE_URL') ||
+    getEnvString('NEXT_PUBLIC_PROMPTFOO_BASE_URL') ||
+    getEnvString('PROMPTFOO_REMOTE_API_BASE_URL') ||
+    `https://api.promptfoo.app`
```

Comment:
This should be hardcoded as a constant and the constant should be used here and in the idempotence check.